src/styles/blocks/caledar.scss

```diff
@@ -0,0 +1,47 @@
+@import "utils/_variables";
+
+.calendar {
+  margin: auto;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gapsize;
+  padding: $calendar-padding;
+  width: $calendar-width;
+
+  &__block {
+    box-sizing: border-box;
+    height: $daysize;
+    width: $daysize;
+    background-color: #eee;
+    border: 1px solid black;
+    transition: 0.25s;
+
+    @for $i from 1 to $days + 1 {
+      &:nth-child(#{$i})::before {
+        content: "#{$i}";
+        display: block;
+        line-height: $daysize;
+        text-align: center;
+      }
+    }
+
+    &:hover {
+      background-color: pink;
+      transform: translateY(-20px);
+      transition: 0.3s;
```

Comment:
![image](https://user-images.githubusercontent.com/96409563/227761020-3139685f-f2c3-4165-9ec4-d12c7f47651f.png)
Also, don't forget to specify transition properties. Now transition applies to all properties, which is not good for performance
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x